department-of-veterans-affairs/next-build
# Fix date format for events

# Description
This addresses [issue 645](https://github.com/department-of-veterans-affairs/next-build/issues/645). We received feedback from the QA analysis to make some changes to how dates are displayed in events. We are still waiting for confirmation that we want to keep datetime localization but we have made the other fillowing changes:

1. Change AM and PM to am and pm
2. Change first comma to a period 

## Ticket
Relates to [645](https://github.com/department-of-veterans-affairs/next-build/issues/645)

## Testing Steps
Visually inspected date listed in events to verify consistency with listings in production. 
Confirm with product stakeholders
Run yarn tests to verify no new issues arrose



## Screenshots
Before:
![image](https://github.com/user-attachments/assets/ef9eed0c-ac75-419b-aa1c-0791bdaf0508)

After:
![image](https://github.com/user-attachments/assets/fe2d1707-0a9c-41b3-8075-2e47d3a648fd)
